content/applications/hr/lunch/management.rst

```diff
@@ -0,0 +1,187 @@
+================
+Lunch management
+================
+
+In Odoo's *Lunch* application, it is required to have someone manage the orders, vendors, and
+products. In addition, someone must be responsible for the orders, and notifying employees when
+their orders have arrived. This can be the same person.
+
+Orders can be :ref:`canceled <lunch/cancel>`, :ref:`sent to the vendor <lunch/send-orders>`,
+:ref:`confirmed <lunch/confirm-orders>` upon arrival, and :ref:`employees can be notified
+<lunch/notify>`, either from the :ref:`Today's Orders <lunch/todays-orders>` dashboard, or the
+:ref:`Control Vendors <lunch/control_vendors>` dashboard.
+
+To manage the *Lunch* app, users need the appropriate :guilabel:`Administrator` rights. These can be
+set by navigating to the :menuselection:`Settings app` and selecting :guilabel:`Manage Users`. Then,
+click on the desired user to view their access rights.
+
+For more information on access rights, refer to the :doc:`Access rights
+</applications/general/users/access_rights/>` documentation.
```

Comment:
Please use a _relative_ path :)
Relative meaning the path **_from_** the doc you are editing **_to_** the doc you are referencing
```suggestion
<../../general/users/access_rights/>` documentation.
```